### PR TITLE
Implement newsletter

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
# Implement Newsletter

_Implement Newsletter

Design and implement a newsletter link at the footer that will enable the users to subscribe using their email address.

Hint: Possibly a button near Join Now on the footer

## Linked issues

fixes https://github.com/EveripediaNetwork/issues/issues/1427

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
@Softdev1 , I added an event logger already to the subscribe/join button

File: src/components/Layout/Footer/Footer.tsx [hunk 1]
```diff
@@ -37,6 +33,11 @@ const Footer = () => {
   const spacing = useBreakpointValue({ base: 8, lg: 24 })
   const [lang, setLang] = useState<string>(languageData[0].value)
   const thisYear = new Date().getFullYear()
+  const newsletterOptions = {
+    bg: '#fff',
+    color: 'gray.800',
+    _hover: { bg: '#fff', color: 'gray.800' },
```
Comment (OleanjiKingCode) [inline src/components/Layout/Footer/Footer.tsx:39]: the hover bg color is the same as the normal bg color

**Comment by Emmanueldmlr** [comment]
@Simplypheyie96 @Softdev1, what's causing the delay in merging this?

**Comment by Simplypheyie96** [comment]
> Visit Preview

@Softdev1 is supposed to present the new design and the design with the input field so we can know which we are passing.

**Comment by Simplypheyie96** [comment]
> @Simplypheyie96 @Softdev1, what's causing the delay in merging this?


Visit Preview

@Softdev1 is supposed to present the new design and the design with the input field so we can know which we are passing.

**Comment by kesar** [comment]
i think you guys are making this incredible complex. it was just adding a button / link in the footer. 🤷🏻‍♂️

**Comment by Emmanueldmlr** [comment]
image

@kesar, that's what we finally did.

**Comment by kesar** [comment]
> image
> @kesar, that's what we finally did.

yup. I am talking about the complexity of process and how long has taken the task to get merged.

5 people involved: 2 devs pushing code, 1 reviewed, ux waiting for product manager, code stale for a week. 

This task can be just add the button and a chat, does it look fine? yes, merged. (1h max work)

**Review by Simplypheyie96 — APPROVED** [review]

**Pull request merged**